out.asciidoc

```diff
@@ -0,0 +1,4 @@
+----
```

Comment:
What is this file for?

Reply:
Thank you for pointing out on it: This file should be auto-generated by test `test_make_setup`. I'll add it to .gitignore.

Reply:
changed to use a temp file